src/Infrastructure/OpcUa/OpcSession.cs

```diff
@@ -317,8 +316,9 @@ private async Task LoadComplexTypeSystemAsync()
             if (_complexTypeSystem == null)
             {
                 _logger.Trace("Loading OPC UA complex type system...");
-                _complexTypeSystem = new ComplexTypeSystem(_session);
-                await _complexTypeSystem.Load();
+                var complexTypeSystemWrapper = new ComplexTypeSystemWrapper(_session);
```

Comment:
I will create an Issue in github so we can use a factory (or something similar) to create an instance of the interface.